admin_scripts/os2displaypc/chromium_setup.sh

```diff
@@ -11,8 +11,9 @@ WIDTH=$3
 HEIGHT=$4
 ORIENTATION=$5
 
-# Setup Chromium user
-useradd chrome -m -p 12345 -s /bin/bash -U
+# Setup Chromium user.
+# useradd will fail on multiple runs, so suppress that error
+useradd chrome -m -p 12345 -s /bin/bash -U || true
```

Comment:
Det ville være meget pænere og ikke mindst mere eksplicit med

```
if [ ! id "chrome" &>/dev/null ]; then
    useradd chrome -m -p 12345 -s /bin/bash -U
fi
```

eller hvad shellcheck nu har det bedst med.

Reply:
Okay tak for review, ændrer det.